src/Test/Utilities/CoreClr/TestUtilities.CoreClr.csproj

```diff
@@ -93,7 +86,4 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(xunitrunnervisualstudioVersion)" />
   </ItemGroup>
   <Import Project="..\..\..\..\build\Targets\Imports.targets" />
-  <PropertyGroup>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.6</NuGetTargetMoniker>
```

Comment:
NuGetTargetMoniker is basically the TFM except if the TFM is .NETCore 5.0 (the UWP\PCL stuff). For a netstandard library NTM == TFM.